moment-timezone.js

```diff
@@ -169,6 +169,11 @@
 					return i;
 				}
 			}
+
+			var alternative = this._getAlternative(timestamp);
```

Comment:
I don't think it makes sense for this logic to be in `_index`, since it's not returning the index for an `until` that represents the timestamp. It would have the correct offset. I think it would be better to create a different method that wraps this information instead, and have the other method call the existing `_index`.